src/buildExec.ts

```diff
@@ -61,9 +61,10 @@ const getToken = async (): Promise<string> => {
       return token;
     } catch (err) {
       setFailure(
-          `Codecov: Failed to get OIDC token with url: ${url}. ${err.message}`,
-          true,
+          `Codecov: Failed to get OIDC token with url, uploading using tokenless: ${url}. ${err.message}`,
+          false,
```

Comment:
I dont think we should silently fail this and go to tokenless. If anything, we should follow the `failCi` [set](https://github.com/codecov/codecov-action/blob/main/src/index.ts#L35).

Reply:
Actually on second thought, I don't know why we don't fail if we don't find OIDC. This is an opt-in feature.

Reply:
this makes sense for public repos that use OIDC for CI that run on branches on that repo, but may also receive forks from external contributors that won't have an id token, and will need to use tokenless

Reply:
@joseph-sentry please see https://github.com/codecov/codecov-action/pull/1404

Reply:
oh, missed that, that works, I'll close this one then!